build_scripts/Debian+Ubuntu/Linux_InstallGuide.md

```diff
@@ -0,0 +1,165 @@
+
+## Compilation on Linux
+
+
+### Install package dependencies:
+#### Debian/Ubuntu
+```bash
+   sudo apt-get install g++ cmake qt5-qmake qtmultimedia5-dev \
+   libqt5x11extras5-dev libbz2-dev libjson-c-dev libssl-dev libsqlcipher-dev \
+   libupnp-dev libxss-dev rapidjson-dev libbotan-2-dev libasio-dev
+```
+
+Additional packages to compile with Qt6:
```

Comment:
It's not exactly "additional". It should be either the qt5 or the qt6 packages. Maybe we should be more clear about that, for instance my making a section for Qt:

sudo apt-get install g++ cmake libbz2-dev libjson-c-dev libssl-dev libsqlcipher-dev \
￼   libupnp-dev libxss-dev rapidjson-dev libbotan-2-dev libasio-dev

then:
To compile with Qt5, add: qt5-qmake qtmultimedia5-dev \
   libqt5x11extras5-dev
To compile with Qt6, add: qt6-base-dev qt6-multimedia-dev qt6-5compat-dev

Is that correct?

Reply:
i will do this separate

Reply:
done